src/pudl_archiver/archivers/epacems.py

```diff
@@ -76,46 +78,35 @@ class EpaCemsArchiver(AbstractDatasetArchiver):
 
     async def get_resources(self) -> ArchiveAwaitable:
         """Download EIA bulk electricity resources."""
-        year_pattern = re.compile(r"\d{4}\/")
-        file_pattern = re.compile(r"(\d{4})([a-z]{2})([0-1][0-9])\.zip")
-
-        # Loop through all available years of data
-        for year in await self.get_hyperlinks(BASE_URL, year_pattern, verify=False):
-            year = int(year[:-1])
-            # Store months available for each state
-            states = {state: [] for state in STATE_ABBREVIATIONS}
-            for link in await self.get_hyperlinks(
-                f"{BASE_URL}/{year}", file_pattern, verify=False
-            ):
-                match = file_pattern.search(link)
-                states[match.group(2)].append(int(match.group(3)))
-
-            for state, months in states.items():
-                if len(months) > 0:
-                    yield self.get_state_year_resource(year, state, months)
+        file_list = requests.get(
+            "https://api.epa.gov/easey/camd-services/bulk-files",
+            params=parameters,
+            timeout=5,
+        )  # FIX ME to use existing classes
```

Comment:
Running this produces "Connection closed" errors and I'm wondering if this line is the source @jdangerx